mff/mff.py

```diff
@@ -776,29 +1111,61 @@ def GenSmoothingMatrix(W,lamstar):
 
 def Reconciliation(y1,W,Phi,C,d,C_ineq=None,d_ineq=None):
     """
-    
+    Reconcile first step forecasts to satisfy equality as well as inequality
+    constraints, subject to smoothening.
 
     Parameters
     ----------
-    y1 : TYPE
-        DESCRIPTION.
-    W : TYPE
-        DESCRIPTION.
-    Phi : TYPE
-        DESCRIPTION.
-    C : TYPE
-        DESCRIPTION.
-    d : TYPE
-        DESCRIPTION.
-    C_ineq : TYPE, optional
-        DESCRIPTION. The default is None.
+    y1 : Series
```

Comment:
Is this a Series or a DataFrame? (I'm not sure)
if this is a Series, let's use `pd.Series` for consistency across the docs.